products/statement-generator/src/components/PrimaryLandingSection.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect } from 'react';
+import { Theme, makeStyles, createStyles } from '@material-ui/core';
+
+import Button from 'components/Button';
+import ContentContainer from 'page-layout/ContentContainer';
+
+interface ComponentProps {
+  goNextPage: () => void;
+}
+
+interface StyleProps {
+    theme?: string;
+}
+
+const useMediaQuery = (query: string) => {
+    const [matches, setMatches] = useState(false);
+
+    useEffect(() => {
+        const media = window.matchMedia(query);
+        if(media.matches !== matches) {
+            setMatches(media.matches);
+        }
+        const listener = () => setMatches(media.matches);
+        window.addEventListener("resize", listener);
+        return () => window.removeEventListener("resize", listener);
+    }, [matches, query]);
+
+    return matches;
+};
+
+const useStyles = makeStyles<Theme, StyleProps>(({ palette, typography, spacing, breakpoints }) =>
+    createStyles({
+        root: {
+            background: palette.secondary.main,
+            color: palette.primary.darker,
+            minHeight: '686px',
```

Comment:
We don't need to match design's height exactly since they don't have the full view of things look at different browser sizes, so let's remove this min-height. Really good job on being meticulous on the design though!